specificatie/genereervariant/openapi.json

```diff
@@ -869,6 +908,145 @@
         "description" : "Geeft aan dat de persoon onder curatele is gesteld.\n",
         "example" : true
       },
+      "GezagPersoonBeperkt" : {
+        "allOf" : [ {
+          "$ref" : "#/components/schemas/PersoonBeperkt"
+        }, {
+          "type" : "object",
+          "properties" : {
+            "gezag" : {
+              "$ref" : "#/components/schemas/Gezag"
+            }
+          }
+        } ]
+      },
+      "Gezag" : {
+        "minItems" : 0,
+        "type" : "array",
+        "items" : {
+          "$ref" : "#/components/schemas/AbstractGezagsrelatie"
+        }
+      },
+      "AbstractGezagsrelatie" : {
+        "required" : [ "type" ],
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string"
+          }
+        },
+        "discriminator" : {
+          "propertyName" : "type",
+          "mapping" : {
+            "TweehoofdigOuderlijkGezag" : "#/components/schemas/TweehoofdigOuderlijkGezag",
+            "EenhoofdigOuderlijkGezag" : "#/components/schemas/EenhoofdigOuderlijkGezag",
+            "GezamenlijkGezag" : "#/components/schemas/GezamenlijkGezag",
+            "Voogdij" : "#/components/schemas/Voogdij",
+            "GezagNietTeBepalen" : "#/components/schemas/GezagNietTeBepalen"
```

Comment:
graag hierbij TijdelijkGeenGezag toevoegen. Deze wordt geleverd als de gezagsmodule soort gezag G geeft. Er is dan wel een minderjarige, geen ouder of derde

Reply:
Toegevoegd